### DISCUSSION TITLE
Storing IPv4 and IPv6 addresses

### DISCUSSION THREAD

**Discussion by andreas83** [prisma/prisma#13260] [Q&A]
I would love to store ip addresses in my mysql database, 
from my research there is the native mysql function INET6_ATON and INET6_NTOA.
is it somehow possible to mix the create object with a raw query part? 

```
prisma.token.create({
       data:{
           ip: 'RAW Function call of INET6_ATON',
           useragent: userAgent,
           token: token,
           created: created,
     }
}
```

**Comment by austincrim** [comment]
Hey @andreas83 👋 ,

It is not possible to mix raw queries and regular queries, but we are aware of some of the use cases where that would be really useful. Feel free to upvote [this issue](https://github.com/prisma/prisma/issues/12279) or add your use case to help us prioritize.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
